cli/tauri.js/package.json

```diff
@@ -104,7 +102,7 @@
     "ts-loader": "8.0.18",
     "typescript": "4.2.3",
     "webpack": "5.26.3",
-    "webpack-cli": "4.5.0",
+    "webpack-cli": "^4.5.0",
```

Comment:
Was this intentional?

Reply:
oops, renovate will fix it lol

Reply:
oh I didn't merge it yet, I'll fix it lol